tests/devices/test_default_mixed_tf.py

```diff
@@ -617,7 +617,7 @@ def cost(params):
         phi = -0.234
         lam = 0.654
 
-        params = tf.Variable([theta, phi, lam], trainable=True)
+        params = tf.Variable([theta, phi, lam], trainable=True, dtype=tf.float64)
```

Comment:
For all the test where you added the precision? Was it failing, if yes we have to document the case and add it to the breaking changes. If no you can ignore this message.

Reply:
Added to breaking changes section and added warning to `finite_diff` if the data is float32.